bin/util.js

```diff
@@ -389,6 +391,7 @@ let downloadVideo = async ({
   }
 
   let downloadUrl = `${qualityUrl}?api_key=${apiKey}`;
+  let showTitle = video.video_show.title;
```

Comment:
Do you happen to know if all Giant Bomb videos include `video_show`?  Mind if we assume it might not exist?

```suggestion
  let showTitle = video.video_show && video.video_show.title ? video.video_show.title : null;
```

Reply:
Ah cool, good spot. Yeah I guess we can't assume that there'll always be a `video_show`. Good to catch it...